.changes/unreleased/Under the Hood-20250117-152215.yaml

```
@@ -0,0 +1,6 @@
+kind: Under the Hood
+body: Misc fixes for group info in logging
+time: 2025-01-17T15:22:15.497485Z
+custom:
+    Author: aranke
+    Issue: ' '
```

You shouldn't leave this blank - you can put the PR number instead when there is no issue.

The changelogs aren't smart enough to not put the links to the issues when this is blank.  It will end up as a broken link in the changelog.  Github treats PRs and issues as the same thing so the links will link to the PR when you sue that.
```suggestion
    Issue: '11218'
```